src/interceptors/ClientRequest/utils/recordRawHeaders.ts

```diff
@@ -46,8 +46,13 @@ export function recordRawFetchHeaders() {
 
       globalThis.Request = OriginalRequest
       globalThis.Response = OriginalResponse
+
+      Object.defineProperty(Headers, kRestorePatches, {
```

Comment:
You can use `Reflect.delete()` and delete the symbol altogether. Deleting it doesn't violate the non-configurable property.

Reply:
@kettanaito Seems like the object must be configurable for `deleteProperity` to work
```js
      console.log(Reflect.deleteProperty(Headers, kRestorePatches)) // returns false
      console.log(Reflect.get(Headers, kRestorePatches)) // return some value (should return undefined)
```

Reply:
Oh, good catch. It's okay to set it configurable from the start so we are able to delete it later.

Reply:
@kettanaito Set it to `configurable`